src/luascript.cpp

```diff
@@ -11942,10 +11942,11 @@ int LuaScriptInterface::luaMonsterTypeGetDefenseList(lua_State* L)
 		return 1;
 	}
 
-	lua_createtable(L, monsterType->spellDefenseList.size(), 0);
+	lua_createtable(L, monsterType->defenseSpells.size(), 0);
 
-	int index = 0;
-	for (const auto& spellBlock : monsterType->spellDefenseList) {
+
+	int32_t index = 0;
```

Comment:
Revert this to int. It's only used to pass the index to lua_rawseti, which takes int as argument.